.github/workflows/docker-image.yml

```
@@ -0,0 +1,89 @@
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+      
+      - name: Build and Push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: src/product-catalog
+          file: Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKER_USERNAME }}/product-catalog-service:${{ github.run_id }}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)